### ISSUE TITLE
Data grid html cell dropdown button aria-label

### ISSUE THREAD

**Issue by nikolab-pn** [telekom/scale#1405]
Scale Version
@telekom/scale-components 3.0.0-beta.113

Framework and version
@telekom/scale-components-react 3.0.0-beta.113

**Current Behavior**
The buttons to open further details about a component are not labelled correctly. The buttons have an aria-label. However, the aria-labelis not on the <button>element that receives the focus, but the element around the button (scale-button). Because of this, the screen readerlists these buttons as unlabeled, which makes it hard for blindusers to perceive their function.

**Expected Behavior**
The aria-label attribute should be moved to the actual button element. I guess it would be good to enable scale button to pass aria-label to button element.

**Desktop (please complete the following information):**
OS: macOS 13.0.1
Browser: Chrome
Version: 107.0.5304.121
